### PR TITLE
pkcs11-helper: fix #47754

### LINKED ISSUE #47754: openvpn with pkcs11 support fails because of pkcs11-helper
I managed to make it work thanks to this [post](http://www.sparklabs.com/forum/viewtopic.php?p=4806#p4823)
I think the solution is to add options in pkcs11-helper's formula and pass them in openvpn's dependency
I'll submit a PR later

### PR DESCRIPTION
**Pull request by xpayn**

### CODE REVIEW

File: Library/Formula/pkcs11-helper.rb [hunk 1]
```diff
@@ -14,6 +14,9 @@ class Pkcs11Helper < Formula
     sha256 "24bb6ab8aa56792d9b658fa617eef363fc9be794dc3327e5afb88eed30ea8af1" => :mavericks
   end
 
+  option "disable-threading", "Build pkcs11-helper without threading support (useful for proper pkcs11 support in openvpn)"
```
Comment (mikemcquaid) [inline Library/Formula/pkcs11-helper.rb]: You can make this sorter to `Build without threading support`. Also, please name it `without-threading`

File: Library/Formula/pkcs11-helper.rb [hunk 2]
```diff
@@ -14,6 +14,9 @@ class Pkcs11Helper < Formula
...
+  option "disable-slotevent", "Build pkcs11-helper without slotevent support (useful for proper pkcs11 support in openvpn)"
```
Comment (mikemcquaid) [inline Library/Formula/pkcs11-helper.rb]: Same with this one.

File: Library/Formula/pkcs11-helper.rb [hunk 3]
```diff
@@ -21,9 +24,15 @@ class Pkcs11Helper < Formula
   depends_on "openssl"
 
   def install
+    args = %W(--disable-debug
```
Comment (mikemcquaid) [inline Library/Formula/pkcs11-helper.rb]: Put the `--disable-debug` and trailing `)` on their own lines.

File: Library/Formula/pkcs11-helper.rb (hunk 1 above)
Comment (xpayn) [inline Library/Formula/pkcs11-helper.rb]: do i need to add a caveat section either in openvpn or pkcs11-helper to explicit the existence of this options?

**Comment by xpayn** [comment]
up
i don't to be pushy, just tell me if i should have waited more before upping

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/5d18c2f5214f793a44a34717ecc93c209be8adfe & https://github.com/Homebrew/homebrew/commit/565ce432c0e2f7564ddd07ba35608a9644a38b7c.

Just a nit but we prefer our args to be wrapped in `[]` rather than `()`. Just a minor style thing.

Thank you for your contributions to Homebrew @xpayn! :smiley_cat:

**Pull request closed without merging**